can-stache-bindings.js

```diff
@@ -957,108 +820,43 @@ var makeDataBinding = function(node, el, bindingData) {
 		bindingInfo.stickyParentToChild && parentObservable,
 		bindingInfo.childEvent,
 		bindingInfo
-	),
-	// these are the functions bound to one compute that update the other.
-	updateParent,
-	updateChild;
-
-	if(bindingData.nodeList) {
-		if(parentObservable) {
-			canReflect.setPriority(parentObservable, bindingData.nodeList.nesting+1);
-		}
+	);
 
-		if(childObservable) {
-			canReflect.setPriority(childObservable, bindingData.nodeList.nesting+1);
-		}
+	// Check for child:from="~parent" (it’s not supported)
```

Comment:
what isn't supported?

Reply:
This comment was wrong and I fixed it: https://github.com/canjs/can-stache-bindings/commit/a0d5124e78c6450822140ac8fe02580fdf83b6ad